en/orm/table-objects.rst

```diff
@@ -140,6 +141,13 @@ Event List
 * ``Model.afterDelete``
 * ``Model.afterDeleteCommit``
 
+initialize
+----------
+
+.. php:method:: initialize(Event $event, ArrayObject $data, ArrayObject $options)
```

Comment:
I think the event should be in the list. I missed it earlier. I think including the signature is fine too. It might be worth noting that table classes do not listen to this event by default and instead use an initialize hook method.